docs/references/basic-transactions.md

```diff
@@ -170,9 +170,16 @@ with the following example.
 ## How to calculate transaction cost
 
 The number of signatures a transaction requires are used to calculate
-the transaction cost. As long as you are not create an account, this
+the transaction cost. As long as you are not creating an account, this
 will be the final transaction cost. To find out more about costs to create
-and account, check out [calculating rent exemption](accounts.md#calculating-rent-exemption)
+an account, check out [calculating rent exemption](accounts.md#calculating-rent-exemption)
+
+<div className="custom-container tip">
+  <h3 className="custom-container title">
+  Deprecation Warning - <code>getRecentBlockhash</code>
+  </h3>
+  <p>Please note that <code>getRecentBlockhash</code> is now deprecated. It is only included in the cookbook for those who still rely on older versions of the web3.js library. It should not be used for version <strong>1.8.0</strong> or higher.</p>
+</div>
```

Comment:
Let's not add this deprecation warning and instead remove the old way of doing it.

Reply:
Okay, I wasn't sure how you would want to handle it. I assumed you would suggest removing the old way altogether, but wanted to err on the side of caution. I'll update this.